client/components/Grid.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import {AgGridColumn, AgGridReact} from 'ag-grid-react';
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
```

Comment:
Ai crap, I forgot to mention that we use [Semantic UI](https://semantic-ui.com/collections/grid.html) in a lot of places in our legacy apps (our new apps are moving towards our internal design system). Might help you get familiar with it to use it here as well? Completely up to you though

Here's the Grid spec for example: https://semantic-ui.com/collections/grid.html

From an exercise POV though, definitely like the AgGrid/AgColumn architecture here, esp field names. Very clean

Reply:
Will checkout semantic ui. Personally not such a huge fan of ag-grid lol